**1) Случайная непрерывная величина A имеет равномерное распределение на промежутке (200, 800]. Найдите ее среднее значение и дисперсию.**

Используем формулы для определения мат. ожидания и дисперсии для случайной непрерывной величины $X$, равномерно распределённой на отрезке $[a,b]$:


Математическое ожидание $M(X) = \frac{a+b}{2}$, Дисперсия $D(X) = \frac{(b-a)^2}{12}$

In [1]:
def mean_and_variance(a,b):
    return f'На промежутке ({a};{b}]\nСреднее значение: М(А) = {(a+b)/2: .2f}\nДисперсия: D(A) = {((b-a)**2)/12: .2f}'
print(mean_and_variance(200,800))

На промежутке (200;800]
Среднее значение: М(А) =  500.00
Дисперсия: D(A) =  30000.00


**Ответ**

$M(A) = 500.00$

$D(A) = 30000.00$

**2)О случайной непрерывной равномерно распределенной величине B известно, что ее дисперсия равна 0.2. Можно ли найти правую границу величины B и ее среднее значение зная, что левая граница равна 0.5? Если да, найдите ее.**

Дисперсия случайной непрерывной равномерно распределённой на отрезке $(a;b)$
величины $B$
 находится по формуле: $D(B) = \frac{(b-a)^2}{12}$


И имея заданные значения $D(B) = 0.2$
 и $a = 0.5$

Найдем правую границу $b$
 величины $B$
 из уравнения $\frac{(b-0.5)^2}{12} = 0.2b-0.5 = \sqrt{2.4b} = 0.5 + \sqrt{2.4}$


И найдем среднее значение по формуле: $M(B) = \frac{a+b}{2}$


In [2]:
b=0.5+2.4**(1/2)
print(f'Левая граница распределения величины В - b = {b: .3f}\n'
      f'Среднее значение В на промежутке (0.5; {b: .3f}) M(B) = {(b+0.5)/2: .3f}'
     )

Левая граница распределения величины В - b =  2.049
Среднее значение В на промежутке (0.5;  2.049) M(B) =  1.275


**Ответ**

Левая граница распределения: $b = 2.049$

Среднее значение: $M(B) = 1.275$



**3)Непрерывная случайная величина X распределена нормально и задана плотностью распределения** $f(x) = (1 / (4 * sqrt(2pi))) * exp((-(x+2)**2) / 32)$

**Найдите:**

а) M(X)

б) D(X)

в) std(X) (среднее квадратичное отклонение)

Нормальное распределение задается функцией плотности вида:
$ f(x) = \frac{1}{\sigma \sqrt{2\pi}} \exp\left(-\frac{(x-\mu)^2}{2\sigma^2}\right) $

Из условий задачи:

$ f(x) = \frac{1}{4 \sqrt{2\pi}} \exp\left(-\frac{(x+2)^2}{32}\right) $

Сравнивая с общей формулой нормального распределения, мы видим, что:

$\sigma = 4$

$\mu = -2$

аким образом, математическое ожидание $M(X) равно:

$M(X) = \mu = -2$

б) D(X) (Дисперсия)
Дисперсия $D(X)$ для нормального распределения равна квадрату стандартного отклонения $σ$:

$D(X) = σ^2 = 4^2 = 16$

в) std(X) (Среднее квадратичное отклонение)
Среднее квадратичное отклонение $std(X)$ равно стандартному отклонению $σ$:

$std(X) = σ = 4$


**Ответ**

$M(X) = -2$

$D(X) = 16$

$std(X) = 4$

**4)Рост взрослого населения города X имеет нормальное распределение. Причем, средний рост равен 174 см, а среднее квадратичное отклонение равно 8 см. Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:**

**а) больше 182 см**

**б) больше 190 см**

**в) от 166 см до 190 см**

**г) от 166 см до 182 см**

**д) от 158 см до 190 см**

**е) не выше 150 см или не ниже 190 см**

**ё) не выше 150 см или не ниже 198 см**

**ж) ниже 166 см.**

Напишем функцию для определения
-значений для данного $Z$ распределения:

In [3]:
def z_value(height):
    return (height-174)/8

In [4]:
from statistics import NormalDist

a) Определим $Z$-значениe:

In [5]:
print(z_value(182))

1.0


метод cdf определяет вероятность того, что случайная величина X будет меньше или равна x, т.е в данном случае $P(X<=182)$

In [6]:
NormalDist().cdf(z_value(182))

0.8413447460685429

Однако нам нужна вероятность $P(X>=182)$, поэтому вычтем полученное значение из $1$:

In [7]:
P=1-NormalDist().cdf(z_value(182))
P

0.15865525393145707

б) Определим $Z$-значениe:

In [8]:
z=z_value(190)
z

2.0

Определим $P(X >=190)$

In [9]:
P=1-NormalDist().cdf(z)
P

0.02275013194817921

в) Определим $Z1$ и $Z2$:


In [10]:
z1=z_value(166)
z1

-1.0

In [11]:
z2=z_value(190)
z2

2.0

Определим $P(166<=X<=190)$

In [12]:
P=NormalDist().cdf(z2)-NormalDist().cdf(z1)
P

0.8185946141203637

г) Определим $Z1$ и $Z2$:


In [13]:
z1=z_value(166)
z1

-1.0

In [14]:
z2=z_value(182)
z2

1.0

Определим $P(166<=X<=182)$

In [15]:
P=NormalDist().cdf(z2)-NormalDist().cdf(z1)
P

0.6826894921370859

д) Определим $Z1$ и $Z2$:

In [16]:
z1=z_value(158)
z1

-2.0

In [17]:
z2=z_value(190)
z2

2.0

Определим $P(158<=X<=190)$

In [18]:
P=NormalDist().cdf(z2)-NormalDist().cdf(z1)
P

0.9544997361036416

е) Определим $Z1$ и $Z2$:

In [19]:
z1=z_value(150)
z1

-3.0

In [20]:
z2=z_value(190)
z2

2.0

Определим $P(X<=150 \cup X >=190)$


In [21]:
P=NormalDist().cdf(z1)+(1-NormalDist().cdf(z2))
P

0.024100029979809312

ё) Определим $Z1$ и $Z2$:

In [22]:
z1=z_value(150)
z1

-3.0

In [23]:
z2=z_value(198)
z2

3.0

Определим $P(X<=150 \cup X >=198)$

In [24]:
P=NormalDist().cdf(z1)+(1-NormalDist().cdf(z2))
P

0.002699796063260207

ж) Определим $Z$-значениe:

In [25]:
z=z_value(166)
z

-1.0

Определим $P(X<=166)$

In [26]:
P=NormalDist().cdf(z)
P

0.15865525393145707

**Ответ**

а)$P(X>=182) = 0.1586$

б)$P(X >=190) = 0.0227$

в)$P(166<=X<=190) = 0.8185$

г)$P(166<=X<=182) = 0.6826$

д)$P(158<=X<=190) = 0.9544$

е)$P(X<=150 \cup X >=190) = 0.0241$

ё)$P(X<=150 \cup X >=198) = 0.0026$

ж)$P(X<=166) = 0.1586$


**5)На сколько сигм (средних квадратичных отклонений) отклоняется рост человека, равный 190 см, от математического ожидания роста в популяции, в которой M(X) = 178 см и D(X) = 25 кв.см?**

Воспользуемся формулой расчета - $Z$ оценки:

$ Z = \frac{X - \mu}{\sigma}  = \frac{190 - 178}{\sqrt{25}} = \frac{12}{5}$

In [27]:
Z=(190-178)/25**(1/2)
Z

2.4

In [28]:
print(f'Рост человека, равный 190 см, отклоняется от математического ожидания роста в популяции, \n'
      f'в которой M(X) = 178 см и D(X) = 25 кв.см на Z = {Z} сигм.')

Рост человека, равный 190 см, отклоняется от математического ожидания роста в популяции, 
в которой M(X) = 178 см и D(X) = 25 кв.см на Z = 2.4 сигм.


**Ответ**

$2.4 сигмы$